packages/application/src/singleWidgetApp.ts

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { JupyterFrontEnd, JupyterFrontEndPlugin } from '@jupyterlab/application';
+
+import { createRendermimePlugins } from '@jupyterlab/application/lib/mimerenderers';
+
+import { LabStatus } from '@jupyterlab/application/lib/status';
+
+import { PageConfig } from '@jupyterlab/coreutils';
+
+import { IRenderMime } from '@jupyterlab/rendermime-interfaces';
+
+import { ISingleWidgetShell, SingleWidgetShell } from './singleWidgetShell';
+
+/**
+ * App is the main application class. It is instantiated once and shared.
+ */
+export class SingleWidgetApp extends JupyterFrontEnd<ISingleWidgetShell> {
+  /**
+   * Construct a new NotebookApp object.
```

Comment:
update for new class name

Reply:
Good catch, fixed in https://github.com/jupyterlite/jupyterlite/pull/498/commits/b98e9a8283e467d93891a225954323dd64fca980